code/modules/admin/verbs/listahelps.dm

```diff
@@ -0,0 +1,307 @@
+	var/uID = "" //The UNIQUE id, made by putting part of the ckey and the ID together. Used internally in code.
+	var/active = "No" //Is the adminhelp active, eg admin responded? This is the same as above, it makes life easier.
+	var/logs = list() //The logs of the adminhelp.
+	var/replying = FALSE //Is someone responding to the adminhelp?
+	var/mob //The mob adminhelping mob.
+
+/client/proc/list_ahelps(user, resolved)
+	if(!check_rights(R_ADMIN))
+		src << "<font color='red'>Error: Only administrators may use this command.</font>"
+		return
+
+	if(resolved)
+		user << "Resolved Ahelps:"
+		for(var/datum/adminticket/T in admintickets)
+			usr << "<span class='adminnotice'><b><font color='red'>#[T.ID] By:</font> <A HREF='?priv_msg=[T.permckey];ahelp_reply=1'>[key_name(T.permuser)]</A></b><b> Ckey:</b> [T.permckey] <b>Name:</b> [T.permuser] <b>Unique ID:</b> [T.uID]</span>"
+			usr << "	<b>Controls:</b> [ADMIN_QUE(mob)] [ADMIN_PP(mob)] [ADMIN_VV(mob)] [ADMIN_SM(mob)] [ADMIN_FLW(mob)] [ADMIN_TP(mob)]"
+			usr << "	<b>Message:</b> [T.msg]"
+			usr << "	<b>Handling Admin:</b> [T.admin]"
+			usr << "	<b>Replied To:</b> [T.active]/<b><a href='?src=\ref[T];view_logs=\ref[T]'>(LOGS)</a></b>"
+			if(T.resolved == "No")
+				usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Resolve)</a>"
+			else
+				usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Unresolve)</a>"
+
+	else
+		user << "Current Unresolved Ahelps:"
+		for(var/datum/adminticket/T in admintickets)
+			if(T.resolved == "No")
+				usr << "<span class='adminnotice'><b><font color=red>#[T.ID] By:</font> <A HREF='?priv_msg=[T.permckey];ahelp_reply=1'>[key_name(T.permuser)]</A></b><b> Ckey:</b> [T.permckey] <b>Name:</b> [T.permuser] <b>Unique ID:</b> [T.uID]</span>"
+				usr << "	<b>Controls:</b> [ADMIN_QUE(mob)] [ADMIN_PP(mob)] [ADMIN_VV(mob)] [ADMIN_SM(mob)] [ADMIN_FLW(mob)] [ADMIN_TP(mob)]"
+				usr << "	<b>Message:</b> [T.msg]"
+				usr << "	<b>Handling Admin:</b> [T.admin]"
+				usr << "	<b>Replied To:</b> [T.active]/<b><a href='?src=\ref[T];view_logs=\ref[T]'>(LOGS)</a></b>"
+				if(T.resolved == "No")
+					usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Resolve)</a>"
+				else
+					usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Unresolve)</a>"
+
+/client/proc/ahelp_count(modifier)
+	var/amount
+	for(var/datum/adminticket/T in admintickets)
+		switch(modifier)
+			if(0)
+				if(T.resolved == "No")
+					amount++
+			if(1)
+				if(T.resolved == "Yes")
+					amount++
+			if(2)
+				amount++
+
+	return amount
+
+
+/datum/adminticket/proc/listtickets()
+	set category = "Admin Help"
+	set name = "List Adminhelps"
+	set desc = "List all current adminhelps"
+
+	if(!check_rights(R_ADMIN))
+		src << "<font color='red'>Error: Only administrators may use this command.</font>"
+		return
+
+	var/count = 0
+
+	for(var/datum/adminticket/T in admintickets)
```

Comment:
No point here, I've tried and it just gives an unused label.  same with the other example of count++